modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/expression/MBLookup.java

```diff
@@ -62,7 +126,17 @@ public MBLookup(JSONArray json) {
      *   ["length", array]: number
      * @return
      */
-    public Expression lookupLength(){ return null;}
+    public Expression lookupLength() {
+        Expression e = parse.string(json,1);
+        if (e.evaluate(e) instanceof String) {
```

Comment:
possibly here as well

Reply:
I left this as is for basic validation for string vs array before returning the appropriate function.